x-pack/metricbeat/module/aws/_meta/docs.asciidoc

```diff
@@ -10,7 +12,7 @@ The default metricsets are `ec2`, `sqs`, `s3_request`, `s3_daily_storage`, `clou
 The `aws` module requires AWS credentials configuration in order to make AWS API calls.
 Users can either use `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and/or
 `AWS_SESSION_TOKEN`, or use shared AWS credentials file.
-Please see <<aws-credentials-config>> for more details.
+Please see <<aws-credentials-config,AWS credentials options>> for more details.
```

Comment:
I wouldn't override the link text here. IMO it's better if the link text matches the section title unless there's a compelling reason to override it.